pallets/creditcoin/src/benchmarking.rs

```diff
@@ -2,7 +2,7 @@
 use super::*;
 
 use crate::benchmarking::alloc::format;
-use crate::helpers::{EVMAddress, PublicToAddress};
+use crate::helpers::{EVMAddress, PublicToAddress, RefstrExt};
```

Comment:
@CAGS295 how do you know that the name is `RefstrExt` ? 

I see that it's got to do with the type names used in the implementation but where is this explained (send me RTFM please)?